File based app does not run

I am Tring to use file based  app example form readme. But looks like `HotReloadHost` is not exposed from  package.

App file:
```
#!/usr/local/share/dotnet/dotnet run
#:package NXUI.Desktop@11.3.0.1
using Avalonia.Controls;
using Avalonia.Styling;
using NXUI.HotReload;
using static NXUI.Builders;

return HotReloadHost.Run(
    () => Window().Content(Label().Content("NXUI")),
    "NXUI",
    args,
    ThemeVariant.Dark,
    ShutdownMode.OnLastWindowClose);
```

Error message:
```
D:\Projects\ConsoleApps\AvaloniaApplication3\UiApp.cs(5,12): error CS0234: The type or namespace name 'HotReload' does not exist in the namespace 'NXUI' (are you missing an assembly reference?)

The build failed. Fix the build errors and run again.
```

--- a/samples/NXUI.Sample.FileApp/App.cs
+++ b/samples/NXUI.Sample.FileApp/App.cs
@@ -1,0 +1,13 @@
+#!/usr/local/share/dotnet/dotnet run
+#:package NXUI.Desktop@12.0.0
+using Avalonia.Controls;
+using Avalonia.Styling;
+using NXUI.HotReload;
+using static NXUI.Builders;
+
+return HotReloadHost.Run(
+    () => Window().Content(Label().Content("NXUI")),
+    "NXUI",
+    args,
+    ThemeVariant.Dark,
+    ShutdownMode.OnLastWindowClose);

--- a/src/NXUI.Desktop/NXUI.Desktop.csproj
+++ b/src/NXUI.Desktop/NXUI.Desktop.csproj
@@ -19,7 +19,11 @@
     <PackageReference Include="Tmds.DBus.Protocol" Version="$(TmdsDbusProtocolVersion)" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="*.props">
+    <Content Include="NXUI.props">
+      <Pack>true</Pack>
+      <PackagePath>build\$(PackageId).props;buildTransitive\$(PackageId).props</PackagePath>
+    </Content>
+    <Content Include="ImplicitUsings.props">
       <Pack>true</Pack>
       <PackagePath>build\;buildTransitive\</PackagePath>
     </Content>

--- a/tests/NXUI.HotReload.Tests/HotReloadHostApiTests.cs
+++ b/tests/NXUI.HotReload.Tests/HotReloadHostApiTests.cs
@@ -1,0 +1,28 @@
+using System;
+using Avalonia.Controls;
+using Avalonia.Styling;
+using Xunit;
+
+namespace NXUI.HotReload.Tests;
+
+public sealed class HotReloadHostApiTests
+{
+    [Fact]
+    public void HotReloadHost_Is_Public_In_Expected_Namespace()
+    {
+        var type = typeof(HotReloadHost);
+
+        Assert.Equal("NXUI.HotReload.HotReloadHost", type.FullName);
+        Assert.True(type.IsPublic);
+        Assert.True(type.IsAbstract);
+        Assert.True(type.IsSealed);
+    }
+
+    [Fact]
+    public void Run_Exposes_File_Based_App_Signature()
+    {
+        Func<Func<object>, string, string[], ThemeVariant?, ShutdownMode, int> run = HotReloadHost.Run;
+
+        Assert.NotNull(run);
+    }
+}

